nservicebus/messaging/databus/azure-blob-storage.md

```diff
@@ -22,19 +22,17 @@ snippet: AzureDataBus
 
 Discarding old Azure DataBus attachments can be done in one of the following ways:
 
-1. Using the built-in method (enabled by default)
-2. Using an Azure Durable Function
-3. Using the Blob Lifecycle Management policy
+partial: cleanup-options
 
 ### Using the built-in clean-up method
 
 Specify a value for the `TimeToBeReceived` property. For more details on how to specify this, see [Discarding Old Messages](/nservicebus/messaging/discard-old-messages.md).
 
-WARN: the built-in method uses continuous blob scanning which can add to the cost of the storage operations. It is **not** recommended for multiple endpoints that are scaled out. If this method is not used, be sure to disable the built-in cleanup by setting the `CleanupInterval` to `0`.
+WARN: the built-in method uses continuous blob scanning which can add to the cost of the storage operations. It is **not** recommended for multiple endpoints that are scaled out. If this method is not used, be sure to disable the built-in cleanup by setting the `CleanupInterval` to `0`. In versions 3 and above built-in cleanup is disabled by default.
 
 ### Using an Azure Durable Function
 
-Review the [sample](/samples/azure/blob-storage-databus-cleanup-function/) to see how to use a durable function to clean up attachments. Be sure to [disable blob cleanup](#configuration-disabling-blob-cleanup) first.
+Review the [sample](/samples/azure/blob-storage-databus-cleanup-function/) to see how to use a durable function to clean up attachments.
```

Comment:
Are you sure you want to remove `Be sure to [disable blob cleanup](#configuration-disabling-blob-cleanup) first.` completely? Perhaps keep it and mention `for version 2 and earlier`?

Reply:
Addressed by https://github.com/Particular/docs.particular.net/pull/4361/commits/708d783121b9c13ecbf8faeb388434a62371b43d